src/services/billing/cost-management/cost-dashboard/cost-dashboard-customize/CostDashboardCustomizePage.vue

```diff
@@ -73,46 +82,98 @@ export default {
         });
 
         const state = reactive({
+            loading: true,
+            layout: [] as CustomLayout[],
+            editingCustomLayout: computed<CustomLayout[]>({
+                get() { return store.state.service.costDashboard.editedCustomLayout; },
+                set(val) {
+                    store.commit('service/costDashboard/setEditedCustomLayout', [...val]);
+                },
+            }),
```

Comment:
```suggestion
            editingCustomLayout: computed<CustomLayout[]>({
                get() { return store.state.service.costDashboard.editedCustomLayout; },
                set(val) {
                    store.commit('service/costDashboard/setEditedCustomLayout', val);
                },
            }),
```

state.editingCustomLayout에 값을 부여할 때 새로운 배열을 만들어 주므로, 여기에서는 새롭게 받은 배열을 그대로 넘겨주면 될 것 같습니다